cmd/verify/verify.go

```diff
@@ -270,8 +255,17 @@ func verifyDigest(record *warc.Record, filepath string, algorithm warc.DigestAlg
 
 func verifyWARCVersion(record *warc.Record, filepath string) (valid bool) {
 	valid = true
+
+	// Check for corrupted version data (indicates WARC reader parsing bug)
```

Comment:
If a version number is ... invalid (or corrupted in the case below) we should likely error on that? At a minimum I feel like a Info would make sense? Seems important.